coap-mbedtls/src/main/java/org/opencoap/transport/mbedtls/DtlsTransportContext.java

```diff
@@ -15,24 +15,41 @@
  */
 package org.opencoap.transport.mbedtls;
 
+import com.mbed.coap.packet.CoapPacket;
+import com.mbed.coap.packet.CoapSerializer;
 import com.mbed.coap.transport.TransportContext;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufOutputStream;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.socket.DatagramPacket;
 import java.time.Instant;
 import java.util.Collections;
 import java.util.Map;
+import java.util.function.BiFunction;
+import org.opencoap.ssl.netty.DatagramPacketWithContext;
 import org.opencoap.ssl.transport.DtlsSessionContext;
 
 public class DtlsTransportContext {
     public static final TransportContext.Key<Map<String, String>> DTLS_AUTHENTICATION = new TransportContext.Key<>(Collections.emptyMap());
     public static final TransportContext.Key<String> DTLS_PEER_CERTIFICATE_SUBJECT = new TransportContext.Key<>(null);
     public static final TransportContext.Key<byte[]> DTLS_CID = new TransportContext.Key<>(null);
     public static final TransportContext.Key<Instant> DTLS_SESSION_START_TIMESTAMP = new TransportContext.Key<>(null);
+    public static final TransportContext.Key<Boolean> DTLS_SESSION_EXPIRATION_HINT = new TransportContext.Key<>(false);
```

Comment:
Is this misleading name? 
Copilot suggested `DTLS_SESSION_PERSISTENCE_HINT`

Reply:
`DtlsSessionContext` from `kotlin-mbedlts` already has property `sessionExpirationHint`, so this change should go also there.

I'm also not sure if `persistence hint` is less misleading as it doesn't imply (for those who don't know) that the session will not be longer available after persistence operation.

The meaning of `session expiration hint` is `hint the underlying transport layer that the session could be prematurely scheduled for expiration`.

Reply:
OK, lets leave it as now, and when we figure out better name then rename it.